### ISSUE TITLE
Camera movement

### ISSUE THREAD

**Issue by partybusiness** [partybusiness/supreme#7]
Adjust camera movement. 

Can tile up and down, can translate up and down, can translate nearer or farther.

**Don't** mess with rotation or translation side-to-side, because the camera direction is used to show what direction the forward axis will move.

**Comment by git-tmurphy57** [comment]
No, it shouldn't; otherwise it might affect gameplay somehow. The heads or any other cosmetic item shouldn't be so big they obstruct or alter gameplay. That includes the camera; which should function exactly the same across cakes.

**Comment by partybusiness** [comment]
Oh, I haven't explained yet. I'm not talking big head mode. When you collect X presents, or whatever you need to finish the level, it unlocks a giant head which lands somewhere in the scene. Then you climb into its mouth to finish the level.

**Comment by partybusiness** [comment]
So I'm thinking the camera should have an animation triggered by entering the mouth.

**Comment by partybusiness** [comment]
![giantheadprototype](https://cloud.githubusercontent.com/assets/22421352/21707893/0b7c7512-d399-11e6-9eb5-a79f86703e63.png)

Giant head prototype

**Comment by partybusiness** [comment]
Maybe it should be wearing a party hat.

**Comment by git-tmurphy57** [comment]
Amazing.

Yes, there should definitely be some kind of animation if this the case. I'm personally imagining some sort of fade to black effect where everything but the cake character fades to black; before using the black screen to transition to something zany and colorful. That's just one idea though. I love the idea of the head.
